thirdparty/README.md

```diff
@@ -339,18 +339,11 @@ File extracted from upstream release tarball:
 ## meshoptimizer
 
 - Upstream: https://github.com/zeux/meshoptimizer
-- Version: 0.15 (2020)
+- Version: git (e4e43fe36e7a8705e602e7ca2f9fb795ded1d0b9, 2020)
 - License: MIT
 
-File extracted from upstream release tarball:
-
```

Comment:
You should keep this part.

Reply:
Changed to `File extracted from upstream tarball:`